cpp/include/rapids_triton/model/shared_state.hpp

```diff
@@ -119,6 +119,22 @@ struct SharedModelState {
     }
   }
 
+  auto get_output_names() const {
```

Comment:
Could you add tests for both this and the other new method? If I'm not mistaken, this will currently return a vector of N empty strings, followed by the actual output names, where N is the number of outputs.

Reply:
As discussed offline, no tests for this yet